ui/src/app/pages/analysis/leonardo-app-launcher.spec.tsx

```diff
@@ -136,63 +143,41 @@ describe('NotebookLauncher', () => {
   });
 
   it('should render', async () => {
-    const wrapper = await notebookComponent();
-    expect(wrapper).toBeTruthy();
+    notebookComponent();
+    expect(
+      screen.getByRole('heading', {
+        name: /loading notebook: wharrgarbl/i,
```

Comment:
I'd prefer a reference to whatever test-data is causing this name to appear

Reply:
Good call. I added the reference.